### ISSUE TITLE
Incorrect formal (total?) charge from PDB file conversion

### ISSUE THREAD

**Issue by openbabel-bot** [openbabel/openbabel#541]
This makes me think, that open-babel does not recognize charges 
inside
PDB files and does not consider the resulting total charge when
calculating partial charges. I also tried to convert a smiles string
containing the for charges into a mol2 file, but again open-babel fails.
So I tried with smiles input. Unfortunately open-babel does not even
create the correct smiles-string from this PDB, but one can help.

Open-babel output:

[N](CC)(CC)(CC)CCOc1cc(C(NCCCCCC(NNc2c3CCCCc3[nH]c3c2cccc3)=
[O-])=[O-])cc(c1OCC[N](CC)(CC)CC)OCC[N](CC)(CC)CC
                                                           !!!!!!!!!!!    

Should be:

[N+](CC)(CC)(CC)CCOc1cc(C(NCCCCCC(NNc2c3CCCCc3[n+H]
c3c2cccc3)=O)=O)cc(c1OCC[N+](CC)(CC)CC)OCC[N+](CC)(CC)CC

The partial charges assigned to a mol2 file created from this add up to
4+, which is good. But, open-babel keeps the order from the smiles
string and one cannot assign them back to the coordinates from the 
PDB.
So, smiles won't do the job!

In summary: for me it seems, that open-babel does not considers 
charges
inside PDB files when converting them.

So, if someone can point me to the right direction, I'd be happy to
hear. What I need is a way to assign correct gasteiger charges to a
structure when converting from PDB to MOL2. The PDB files carry a 
total
charge!

Paul

Reported by: @ghutchis
